src/shared/components/TruncatedText.tsx

```diff
@@ -1,12 +1,13 @@
 import * as React from 'react';
-import {If, Else, Then} from 'react-if';
+import classNames from 'classnames';
```

Comment:
oops, this was a `copy&paste` from another file, apparently we have it like this at many places already.